src/express.js

```diff
@@ -2,7 +2,7 @@
 
 import https from 'https';
 
-import express from 'express';
+import express from 'express'; // eslint-disable-line import/no-unresolved
```

Comment:
this has been failing for a long time. its a peer dep so eslint cant find it in import resolution. lets just ignore